packages/itwinui-react/src/core/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -26,6 +26,11 @@ export type ToggleSwitchProps = {
    * Icon inside the toggle switch. Shown only when toggle is checked.
    */
   icon?: JSX.Element;
+  /**
+   * Size of the toggle switch.
+   *  @default 'default'
+   */
+  switchSize?: 'default' | 'small';
```

Comment:
The `switchSize` name is a bit weird. Maybe we should use `size`? Only concern there is potential clash with the [native `size`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/size) attribute, but that is not applicable to checkbox inputs, so we should be fine.